tests/test_box_mesh.py

```diff
@@ -134,41 +158,73 @@ def _check_triangle_on_boundary(
     return False
 
 
-def test_given_box_mesh__construct_must_find_center_corners_edges_faces_node_sets(
-    default_box_mesh,
+@pytest.mark.parametrize(
+    "box_mesh",
+    [
+        "default_box_mesh",
+        "expanded_box_mesh_x",
+        "expanded_box_mesh_y",
+        "expanded_box_mesh_z",
+    ],
+)
+def test_given_box_mesh_construct_must_find_center_corners_edges_faces_node_sets(
+    box_mesh: BoxMesh, request
 ) -> None:
-    target_center: npt.NDArray[np.float_] = np.array([0.5, 0.5, 0.5])
+    box_mesh_to_test, rve = request.getfixturevalue(box_mesh)
+    target_center: npt.NDArray[np.float_] = rve.center
     target_corners: List[int] = [0, 2, 6, 8, 18, 20, 24, 26]
     target_edges: List[int] = [1, 3, 5, 7, 9, 11, 15, 17, 19, 21, 23, 25]
     target_faces: List[int] = [4, 10, 12, 14, 16, 22]
 
-    box_mesh_corners = sorted(np.concatenate(list(default_box_mesh.corners.values())))
-    box_mesh_edges = sorted(np.concatenate(list(default_box_mesh.edges.values())))
-    box_mesh_faces = sorted(np.concatenate(list(default_box_mesh.faces.values())))
+    box_mesh_corners = sorted(np.concatenate(list(box_mesh_to_test.corners.values())))
+    box_mesh_edges = sorted(np.concatenate(list(box_mesh_to_test.edges.values())))
+    box_mesh_faces = sorted(np.concatenate(list(box_mesh_to_test.faces.values())))
 
     assert (
-        (default_box_mesh.center == target_center).all()
+        (box_mesh_to_test.center == target_center).all()
         and box_mesh_corners == target_corners
         and box_mesh_edges == target_edges
         and box_mesh_faces == target_faces
     )
 
 
-def test_given_box_mesh_rve_property_must_build_correct_rve(default_box_mesh) -> None:
-    target_rve = _default_rve()
-    test_rve = default_box_mesh.rve
+@pytest.mark.parametrize(
+    "box_mesh",
+    [
+        "default_box_mesh",
+        "expanded_box_mesh_x",
+        "expanded_box_mesh_y",
+        "expanded_box_mesh_z",
+    ],
+)
+def test_given_box_mesh_rve_property_must_build_correct_rve(
+    box_mesh: BoxMesh, request

```

Comment:
`request` parameter should be typed as `FixtureRequest`

`from pytest import FixtureRequest`